src/sc2utils/sc2_manage_process.cc

```diff
@@ -238,7 +238,7 @@ bool TerminateProcess(uint64_t process_id) {
     if (index < 0)
         return false;
 
-    ::TerminateProcess(windows_processes[index].pi_.hProcess, -1);
+    ::TerminateProcess(windows_processes[index].pi_.hProcess, static_cast<UINT>(-1));
```

Comment:
Is `UINT` an MSVC only type? It doesn't compile with clang or gcc. Why not static cast to the actual argument type `uint64_t`?